wsgi.py

```diff
@@ -49,13 +49,15 @@ def get_openshift():
 
 @AUTH.verify_password
 def verify_password(username, password):
-    with open("/app/auth/users", "r") as my_file:
-        user_str = my_file.read()
-    if user_str:
-        user = user_str.split(" ", 1)
-        if username == user[0] and password == user[1]:
-            return username
-    return None
+    try:
+        with open("/app/auth/users", "r") as my_file:
+            user_str = my_file.read()
+        if user_str:
+            user = user_str.split(" ", 1)
+            if username == user[0] and password == user[1]:
+                return username
+    except FileNotFoundError:
+        return True
```

Comment:
Should this be part of a separate PR?

Reply:
No, as it's needed for the tests. I saw that the testing script wasn't providing authentication, so this implements the option of not having api authentication via basic auth.